src/org/components/EventForm/steps/OrganizerStep.tsx

```diff
@@ -287,86 +287,53 @@ export const OrganizerStep = ({
           </FormSection>
           {!isNotOnWeb && (
             <>
-              <FormSection header="Kontaktní osoba" required onWeb>
```

Comment:
Ještě tady. Možná by se to nemělo mazat, ale přidat tam používání kontaktních údajů místo kontaktní osoby.

Protože flow má zůstat stejný:
"Mohu vybrat "stejná jako hlavní organizátor" nebo člověka z selectu. Jeho údaje se propíšou do kontaktních údajů. Ty by si ale měl mít možnost upravit, přepsat atd. Zároveň by mělo být možné kontaktní údaje vepsat ručně bez toho, aniž bych vybrala konkrétního uživatele z selectu nebo zaškrtla "stejná jako hlavní organizátor". (od Hanky)

Reply:
@talitadzikzlasu jo, s tím souhlasím, že by to bylo lepší, ale to je aktuálně trochu mimo můj čas a sílu. Spíš jsem si to chtěl jen zkusit, ze zajímavosti, jak se s tím pracuje, a teda taky proto, že protože už jsem tu změnu nasadil na dev, tak kdybych ji chtěl vrátit zpět, tak zas pro změnu musím řešit migrace atp.

Reply:
Ano, rozumím.. Ale přijde mi špatné to teď mazat a potom to znovu psát. Tak navrhuji buď udělat to v tom pull requestu, nebo zakomentovat ten kód, který ještě možná bude použit, a přidat todo a issue.

Reply:
zakomentování použitelného kódu zní jako dobrý kompromis 🙂

Reply:
Some of the code kept around in 9409c5da11f52e031467930f3b337c52cf2dd26e